src/main/java/frc/robot/utils/dynamicauto/ChoreoBuilder.py

```diff
@@ -0,0 +1,117 @@
+import json
+
+# Assuming locations is a list of dictionaries where each dictionary represents a location
+# L,M, and R were change to S1, S2, and S4 with an S3 being added.
+locations = [
+    {"name": "Amp Side", "x": 0.71, "y": 6.72, "angle": 1.04, "allowed_destinations": ["W1", "W2", "W3", "C1", "C2", "C3", "C4", "C5"]},
+    {"name": "Center", "x": 1.35, "y": 5.56, "angle": 0, "allowed_destinations": ["W1", "W2", "W3", "C1", "C2", "C3", "C4", "C5"]},
+    {"name": "Stage Side", "x": 0.71, "y": 4.36, "angle": -1.04, "allowed_destinations": ["W1", "W2", "W3", "C1", "C2", "C3", "C4", "C5"]},
+    {"name": "W1", "x": 2.3, "y": 6.757, "angle": 0.39, "allowed_destinations": [ "W2", "W3", "S1", "S2", "S3", "S4", "C1", "C2", "C3", "C4", "C5"]},
```

Comment:
consider making the wing notes aligned to the speaker so you can shoot directly from them